src/screens/MapScreen.tsx

```diff
@@ -42,13 +50,15 @@ const MapScreen: React.FC<MapScreenProps> = ({ toggleIsStrolling }: MapScreenPro
 					<SelectionBox selections={selections} setSelections={setSelections} />
 				) : null}
 				<StrollButton
-					onStartStrolling={toggleIsStrolling}
+					onStartStrolling={() => setPageStrolling("strollScreen")}
```

Comment:
this should just be `onStartStrolling={setPageStrolling}` - see the longer comment above